assets/app/styles/_components.less

```diff
@@ -0,0 +1,269 @@
+          text-align: right;
+          .align-self(@align: center)
+        }
+        &.pod-template-block {
+          text-align: left;
+          display: inline-block;
+          .flex(@columns: 1 0 auto);
+          max-width: 450px;
+          border-radius: 3px;
+          margin-bottom: 10px;
+          &:nth-of-type(odd) {
+            margin-right: 10px;
+          }
+          .pod-template {
+            border: 1px dotted #ddd;
+            border-radius: @border-radius-md;
+            padding: 5px 10px;
+            h4 {
+              margin: 5px 0;
+            }
+            .fa {
+              margin-right: 5px;
+            }
+          }
+          .pod-template-ports {
+            .font-icon-block {
+              margin-right: 8px;
+            }
+            .port-number {
+              display: inline-block;
+            }
+          }
+        }
+      }
+    }
+    &.service {
+      border-left-width: 0;
+      margin-bottom: 6px;
+      padding: 6px 15px 5px;
+      .osc-object-highlight();
+      &.none {
+        background-color: transparent;
+      }
+    }
+    &.deployment-block {
+      margin: 0 0 0 6px;
+      padding: 5px 15px;
+      &.none {
+        background-color: transparent;
+        &:after {
+          background-color: #ccc;
+          top: 50%;
+          content: "";
+          display: block;
+          height: 1px;
+          left: -10px;
+          position: absolute;
+          width: 9px;
+        }
+      }
+    }
+  }
+}
+
+.builds-block {
+  margin-left: 5px;
+  .builds {
+    margin: 0;
+    overflow: hidden;
+    .build {
+      border-radius: @border-radius-md;
+      padding: 5px;
+      position: relative;
+      margin: 10px 0;
+      text-align: left;
+      &:first-child {
+        margin-top: 0;
+      }
+      &:after {
+        background-color: @connector-color;
+        bottom: -5px;
+        content: "";
+        display: block;
+        height: 1px;
+        left: 0;
+        right: 0;
+        position: absolute;
+      }
+    }
+  }
+}
+
+
+.pod-block {
+  display: block;
+  .pod-container {
+    .flex-display(@display: flex);
+    .flex-direction(@direction: row);
+    .flex-wrap(@wrap: wrap);
+    .justify-content(@justify: flex-start);
+    margin-top: 5px;
+    &.pod-container-lg {
+      .pod {
+        width: 70px;
+        height: 70px;
+      }
+    }
+    .pod {
+      .osc-object-highlight();
+      .flex(@columns: 1 1 auto);
+      max-width: 450px;
+      border-radius: @border-radius-md;
+      border: 1px solid transparent;
+      // box-shadow: @osc-object-box-shadow; <- remove if it's not used in highlight
+      padding: 10px;
+      position: relative;
+      margin: 0 10px 10px 0;
+      text-align: center;
+      background-color: @pod-pending-bg;
+      -webkit-font-smoothing: antialiased;
+      &.pod-running {
+        background-color: @pod-running-bg;
+        .spinner {
+          display: none;
+        }
+      }
+      &.pod-failed {
```

Comment:
@sg00dwin We already talked about this, but we need separate pod-failed and pod-warning styles.